src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/Obsolete/ErrorPage.cshtml

```diff
@@ -0,0 +1,262 @@
+﻿@using System
```

Comment:
We want these files to be present for users who _might_ be using these types. In later major release we can just delete the whole folder.